apps/system/iperf/iperf_udp.c

```diff
@@ -50,6 +50,8 @@
 #include <sys/socket.h>
 #include <sys/types.h>
 #include <netinet/in.h>
+#include <arpa/inet.h>
+
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?